Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN apt-get update && \
       ros-$ROS_DISTRO-rmw-fastrtps-cpp \
       ros-$ROS_DISTRO-rmw-connextdds \
       ros-$ROS_DISTRO-rmw-cyclonedds-cpp \
-    && pip3 install \
+    && pip3 install --break-system-packages \
```

Comment:
because of https://peps.python.org/pep-0668/. Maybe the cleaner way is to use venv

Reply:
? what happens if we don't include this?

Reply:
This: https://github.com/open-navigation/navigation2/actions/runs/8858102523/job/24326368742

Reply:
Yea, I encountered this as well. it's the simplests workaround until colcon org upstreams my cache extension.
They really want folks to sign a waiver agreement now before handing over a :foot: :gun: .